src/lib.rs

```diff
@@ -275,8 +275,11 @@ impl Config {
     /// Registers a dependency for this compilation on the native library built
     /// by Cargo previously.
     ///
-    /// This registration will modify the `CMAKE_PREFIX_PATH` environment
-    /// variable for the build system generation step.
+    /// This registration will add the path in the environment
+    /// variable `DEP_XXX_ROOT`, where `XXX` is replaced with the
+    /// uppercased value of `dep`, to the `CMAKE_PREFIX_PATH`
+    /// environment variable for the [`build`][Self::build] system
+    /// generation step.
```

Comment:
This is a bit tricky, maybe better to split to two sentences. E.g.:

```rust
/// This will update the `CMAKE_PREFIX_PATH` environment variable for the [`build`][Self::build]
/// system generation step. The path will be updated to include `DEP_XXX_ROOT`, where `XXX`
/// is replaced by the uppercased value of `dep`.
```

Also most of the other doc comments use line length 80, could you reflow to that?

Reply:
I've pushed a new version following your suggestions.